audiobook_generator/book_parsers/epub_book_parser.py

```diff
@@ -45,7 +45,12 @@ def get_chapters(self, break_string) -> List[Tuple[str, str]]:
         for item in self.book.get_items_of_type(ebooklib.ITEM_DOCUMENT):
             content = item.get_content()
             soup = BeautifulSoup(content, "lxml")
-            title = soup.title.string if soup.title else ""
+            title = ""
+            title_levels = ['title', 'h1', 'h2', 'h3']
```

Comment:
I'm a bit concerned about the possibility that items labeled as h1 h2 h3 could be non section title. However, it's not a big issue, and if there is indeed a problem, we can fix it later.

Reply:
> I'm a bit concerned about the possibility that items labeled as h1 h2 h3 could be non section title. However, it's not a big issue, and if there is indeed a problem, we can fix it later.

Where did you get your audio book? I have converted my favorite two books, each page's `<head>` element content was somehow cleared, though, their section info is designed to be in the h1 or h2 element, so I still can get the exact section title, in the past code, I get the first 100 characters.

soup sample: 
```
<?xml version='1.0' encoding='utf-8'?><!DOCTYPE html>
<html epub:prefix="z3998: http://www.daisy.org/z3998/2012/vocab/structure/#" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops">
<head></head>
<body>
<h2>Section XX</h2>
```